packages/data-context/src/actions/WizardActions.ts

```diff
@@ -321,17 +321,23 @@ interface E2eScaffoldOpts {
 }
 
 const E2E_SCAFFOLD_BODY = (opts: E2eScaffoldOpts) => {
+  let e2eFields = `specPattern: 'cypress/e2e/**/*.cy.{js,ts}',`
+
+  if (opts.lang === 'ts') {
+    e2eFields = `supportFile: 'cypress/support/e2e.ts',
```

Comment:
Why is this default support file value added if `ts` is used? Would it be reasonable to assume the default when TS is not provided to be `cypress/support/e2e.js`?

Reply:
I am a little confused too. What does this PR actually do? It says "remove default supportFile" it's more like "only have a default `supportFile` is the lang is `ts` isn't it?

@estrada9166 can you clarify what this PR is doing?

Reply:
Looks like we can close in favor of https://github.com/cypress-io/cypress/pull/19776/files maybe?